### DISCUSSION TITLE
Room capture

### DISCUSSION THREAD

**Discussion by luk6173** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#380] [Q&A] [answered]
Hi all,

how can I capture the room exactly as it appears in the picture?
![grafik](https://user-images.githubusercontent.com/24668586/163186686-bed1f079-e928-49ef-805f-115bcf40da88.png)

Thank you & Regards!

**Comment by PiotrMachowski** [comment, marked as answer]
You have to create an outline: https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/318
